dockers/docker-orchagent/supervisord.conf

```diff
@@ -71,6 +71,18 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=orchagent:running
 
+[program:coppmgrd]
+command=/usr/bin/coppmgrd
```

Comment:
Based on [HLD](https://github.com/Azure/SONiC/blob/master/doc/copp/CoPP%20Config%20and%20Management.md#coppmgr), coppmgrd shall be started before other processes. Is there any reason for starting at priority 6?

Reply:
Its starting immediately after the orchagent. The initial idea was to start coppmgrd at the same priority as start.sh which was earlier programming copp